Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM python:3.5.5-jessie
+
+# TODO: move in a base image
+RUN useradd -ms /bin/bash -G www-data elife && \
+    chown elife:elife /srv && \
+    mkdir /srv/bin && \
+    chown elife:elife /srv/bin && \
+    mkdir -p /var/www && \
+    chown www-data:www-data /var/www
+
+RUN pip install virtualenv
+RUN pip install -U setuptools
+# END base image
+
+USER elife
+ENV PROJECT_FOLDER=/srv/profiles
+RUN mkdir ${PROJECT_FOLDER}
+WORKDIR /srv/profiles
+COPY --chown=elife:elife install.sh requirements.txt /srv/profiles/
+RUN PROFILES_SKIP_DB=1 /bin/bash install.sh
+
+## yes this is how you copy directories
+COPY --chown=elife:elife manage.py /srv/profiles/
+COPY --chown=elife:elife migrations /srv/profiles
+COPY --chown=elife:elife profiles/ /srv/profiles/profiles
```

Comment:
omitting tests/ and similar: that can be done in an extension `profiles_ci` image (that is never published)